content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -18,130 +18,118 @@ endif::[]
 
 = Pipeline Editor
 
-The Blue Ocean Pipeline Editor is the simplest way for anyone to get started with
-creating Pipelines in Jenkins. It's also a great way for existing Jenkins users
-to start adopting Pipeline.
+The Blue Ocean Pipeline Editor is an easy way for anyone to get started with creating Pipelines in Jenkins.
+It's also great for existing Jenkins users to start adopting Pipeline.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
-The editor allows users to create and edit Declarative Pipelines, add
-stages, and parallelized tasks that can run at the same time, depending on their
-needs. When finished, the editor saves the Pipeline to a source code repository
-as a `Jenkinsfile`.  If the Pipeline needs to be changed again,
-Blue Ocean makes it easy to jump back into the visual editor to modify the
-Pipeline at any time.
+The editor allows users to create and edit Declarative Pipelines and perform actions such as adding stages or configuring parallel tasks, depending on their needs.
+When the user completes their configuration, the editor saves the Pipeline to a source code repository as a `Jenkinsfile`.
+If the Pipeline requires further modification, Blue Ocean makes it easy to jump back into the visual editor to modify the Pipeline at any time.
 
 video::FhDomw6BaHU[youtube, width=640, height=360, align="center"]
 
-== Starting the editor
+[NOTE]
+.Blue Ocean Pipeline Editor limitations
+====
+The Blue Ocean Pipeline Editor is a great alternative for Jenkins users, but there are some limitations to its functionality:
 
-To use the editor a user must first have
-<<creating-pipelines, created a pipeline in Blue Ocean>>
-or have one or more existing Pipelines already created in Jenkins.
-If editing an existing pipeline, the credentials for that pipeline must allow pushing of
-changes to the target repository.
+* The editor utilizes SCM-based Declarative Pipelines only.
+* User credentials must have write permission.
+* The editor does not have full parity with Declarative Pipeline.
+* The Pipeline is re-ordered and comments are removed.
+====
 
-The editor can be launched via:
+== Starting the editor
 
-* Dashboard "New Pipeline" button
-* Activity View for Single Run
-* Pipeline Run Details
+To use the editor, a user must first <<creating-pipelines, create a Pipeline in Blue Ocean>> or have at least one existing Pipeline in Jenkins.
+If editing an existing Pipeline, the credentials for that Pipeline must allow pushing changes to the target repository.
 
-== Limitations
+The editor can be launched through the:
 
-* SCM-based Declarative Pipelines only
-* Credentials must have written permission
-* Does not have full parity with Declarative Pipeline
-* Pipeline re-ordered and comments removed
+* *New Pipeline* option from the link:https://www.jenkins.io/doc/book/blueocean/dashboard/[Blue Ocean dashboard].
+* *Branches* tab within the link:https://www.jenkins.io/doc/book/blueocean/activity/#branches[Activity view].
```

Comment:
It is better for those developing content for the site if the links are "site local" (without the https://www.jenkins.io prefix) so that while developing locally I am not taken back to www.jenkins.io through a clicked link
```suggestion
* *New Pipeline* option from the link:/doc/book/blueocean/dashboard/[Blue Ocean dashboard].
* *Branches* tab within the link:/doc/book/blueocean/activity/#branches[Activity view].
```